.changes/2.0.1.md

```diff
@@ -0,0 +1,163 @@
+# [2.0.1] (2022-10-12)
```

Comment:
I think this changelog section is for 2.0.2 and the 2.0.3 changelog is missing: https://github.com/hashicorp/setup-terraform/releases

For 2.0.3, the human readable changelog can be something like:

## NOTES

* Reduced occurrences of GitHub Actions warnings for setting output [#247](https://github.com/hashicorp/setup-terraform/pull/247)